homeassistant/components/overkiz/const.py

```diff
@@ -59,6 +61,11 @@
     UIClass.POD,
 ]
 
+OVERKIZ_REFRESH_DEVICE_STATES_DEVICES: list[UIClass | UIWidget] = [
+    UIClass.GARAGE_DOOR,
+    UIClass.GATE,
+]
```

Comment:
Could we make this more specific? I am not sure if an UIClass (or widget) is specific enough. There are many many Overkiz devices and perhaps we should scope this on the `controllable_name`?

Reply:
Sure, makes sense.

Reply:
Could we add the cover ?
My RS100 io don't push there state when using the wired command